dags/hindawi/xml_extractors.py

```diff
@@ -20,7 +20,7 @@ def _get_extracted_value(self, node):
             return self.default_value
         pattern = re.compile(r"[\n\t]*")
         extracted_value = "".join(
-            [pattern.sub("", node.text) or ""]
+            [node.text and pattern.sub("", node.text) or ""]
```

Comment:
could you please add a test? Also it's getting too complex, could you please separate the logic?

Reply:
@ErnestaP ☝️